libraries/resource_provider.rb

```diff
@@ -19,35 +19,67 @@ class ResourceProvider
       :item,          # The OneviewSDK resource instance
       :sdk_api_version,  # Keep track of the API version used for the SDK resource class
       :sdk_variant,      # Keep track of the variant used for the SDK resource class
-      :sdk_resource_type # Keep track of the SDK resource class name
+      :sdk_resource_type, # Keep track of the SDK resource class name
+      :sdk_secondary_api  # Keep track of the internal secondary API used by the SDK
```

Comment:
Great, I had was the same thought on naming it `base_sdk` but I guessed it could be confusing, but it's prettier... I'll go with it.